core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpSamDAO.scala

```diff
@@ -127,7 +127,7 @@ class HttpSamDAO(baseSamServiceURL: String, serviceAccountCreds: Credential)(imp
   }
 
   override def getUserStatus(userInfo: UserInfo): Future[Option[RawlsUser]] = {
-    val url = samServiceURL + "/register/user/v2/info"
+    val url = samServiceURL + "/register/user/v2/self/info"
```

Comment:
This endpoint was incorrect, so we were always 404ing and not exercising the logic on L30 of [StandardUserInfoDirectives](core/src/main/scala/org/broadinstitute/dsde/rawls/openam/StandardUserInfoDirectives.scala)